examples/embassy-net-udp/README.md

```diff
@@ -13,3 +13,5 @@ In this folder, run
 
 With the device USB cable connected, a USB ethernet device should pop up.
 RIOT-rs will reply to ping requests on 10.42.0.61.
+
```

Comment:
not sure if this is absolutely needed but is the ping -I <DEV> option needed here ? (If so add this precision for beginners?)

Reply:
`-I` shouldn't be needed?